library/src/main/java/com/devbrackets/android/exomedia/core/video/ResizingTextureView.java

```diff
@@ -331,9 +338,7 @@ public void setVideoRotation(@IntRange(from = 0, to = 359) int userRotation, @In
         requestedUserRotation = userRotation;
         requestedConfigurationRotation = configurationRotation;
 
-        if (matrixManager.ready()) {
-            matrixManager.rotate(this, (userRotation + configurationRotation) % MAX_DEGREES);
-        }
+        updateRotateScale();
```

Comment:
Revert this to `matrixManager.rotate(...)` but without the `.ready()` check.  That should be moved in to the `MatrixManager` itself. (i.e. in rotate, if we are ready apply it, otherwise store the requested value and update when we are ready)